front/src/common/BootstrapSNCF/FormSNCF/DebouncedNumberInputSNCF.tsx

```diff
@@ -33,7 +33,7 @@ const DebouncedNumberInputSNCF = ({
     else if (value === null && input !== 0) setInput(0);
   };
 
-  useDebouncedFunc(value, debouncedDelay, checkChangedInput);
+  useDebouncedFunc(value as number, debouncedDelay, checkChangedInput);
```

Comment:
You can remove the cast here, but you need also to change the signature of checkChangedInput (number | null)

```suggestion
  useDebouncedFunc(value, debouncedDelay, checkChangedInput);
```

Reply:
need to talk about it